adapters/appnexus/appnexus.go

```diff
@@ -85,15 +85,10 @@ func (a *adapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapters.E
 	memberIds := make(map[string]bool)
 	errs := make([]error, 0, len(request.Imp))
 
-	// AppNexus openrtb2 endpoint expects imp.displaymanagerver to be populated, but some SDKs will put it in imp.ext.prebid instead
-	var defaultDisplayManagerVer string
-	if request.App != nil {
-		source, err1 := jsonparser.GetString(request.App.Ext, openrtb_ext.PrebidExtKey, "source")
-		version, err2 := jsonparser.GetString(request.App.Ext, openrtb_ext.PrebidExtKey, "version")
-		if (err1 == nil) && (err2 == nil) {
-			defaultDisplayManagerVer = fmt.Sprintf("%s-%s", source, version)
-		}
-	}
+	// appnexus adapter expects imp.displaymanagerver to be populated in openrtb2 endpoint
+	// but some SDKs will put it in imp.ext.prebid instead
+	defaultDisplayManagerVer := buildDefaultDisplayManageVer(request)
```

Comment:
nitpick - I think we should not call this defaultDisplayManagerVer since we are building it using `req.App.Ext`. Should we just call it - `displayManagerVer`. This would also be inline with openrtb2 spec - `imp.displaymanagerver`

Reply:
https://github.com/prebid/prebid-server/commit/0a16005e7f948517fd599b62c4f882d0334f3b6b makes the changes